Networking/NetworkingTests/Remote/ShippingLabelRemoteTests.swift

```diff
@@ -190,4 +226,13 @@ private extension ShippingLabelRemoteTests {
                                     city: "SAN FRANCISCO",
                                     postcode: "94110-4929")
     }
+
+    func sampleShippingLabelCustomPackage() -> ShippingLabelCustomPackage {
+        return ShippingLabelCustomPackage(isUserDefined: true,
+                                                       title: "Test Package",
+                                                       isLetter: false,
+                                                       dimensions: "10 x 10 x 10",
+                                                       boxWeight: 5,
+                                                       maxWeight: 1)
```

Comment:
nit: fix the alignment of the parameters 👍

Reply:
Thanks for catching that! Fixed in 58c42be.